SPECS-EXTENDED/liblouis/liblouis.spec

```diff
@@ -0,0 +1,407 @@
+# Turn off the brp-python-bytecompile script
+%global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
+Summary:        Braille translation and back-translation library
+Name:           liblouis
+Version:        3.22.0
+Release:        2%{?dist}
+License:        LGPLv3+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            http://liblouis.org
```

Comment:
Can we change this to point somewhere else? Judging by the registration history of the URL, they're in the middle of moving domain registrars and this currently leads to an ad-infested registrar page.

Reply:
I have modified the URL to the following. Hope that is fine. 
https://github.com/liblouis/liblouis